tools/cloud-build/daily-tests/builds/ctk-dockerfile.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+tags: [dockerfile, m.pre-existing-vpc, m.vm-instance, vm]
+
+timeout: 3600s  # 1hr
+steps:
+## uses a pre-built Google Cloud image containing Docker CLI tool.
+- id: build-docker-image
+  name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcluster', 'tools/cloud-build/images/cluster-toolkit-dockerfile/']
+
+- id: create-deployment-folder
+  name: 'gcluster'
```

Comment:
This looks like we're calling the `gcluster` binary instead of running it in the Docker container.  Could you update this to run in the Docker container?

Reply:
```
- id: create-deployment-folder
  name: 'gcluster'
```
It's already running in a Docker container. Here, `gcluster` is the Docker image we built in the preceding step. Cloud build uses the `gcluster` image to run the docker container. Cloud build automatically runs each step as a separate container as an instance of `docker run`. More details [here](https://cloud.google.com/build/docs/build-config-file-schema#:~:text=A%20build%20step%20specifies%20an,steps%20in%20your%20config%20file.&text=args:%20%5B'build'%2C,'%5D&text=%22build%22%2C,%22.%22)

Reply:
My mistake, I thought the "name" field was to give the step a name!  Thanks for the explanation and the link to docs!